src/components/Dashboard/DashboardCard/DashboardCardTitleSeeAll.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, OverlayTrigger, Popover } from 'patternfly-react';
+
+const SEE_ALL = 'See all';
+
+const DashboardCardTitleSeeAll = ({ title, children }) => {
+  if (React.Children.count(children) === 0) {
+    return null;
+  }
+  const overlay = (
+    <Popover id="popover" title={title}>
+      {children}
+    </Popover>
+  );
+  return (
+    <OverlayTrigger overlay={overlay} placement="right" trigger={['click']} rootClose>
+      <Button bsStyle="link">{SEE_ALL}</Button>
+    </OverlayTrigger>
+  );
+};
+
+DashboardCardTitleSeeAll.defaultProps = {
+  children: null,
+};
+
+DashboardCardTitleSeeAll.propTypes = {
+  children: PropTypes.node,
+  title: PropTypes.string.isRequired,
+};
+
+export default DashboardCardTitleSeeAll;
```

Comment:
can you remove this default export and export DashboardCardTitleSeeAll itself ? we usually do not use default exports

Reply:
Sure :+1: